File: Playground/fizzbuzz.py
def fizzbuzz(n):
    for i in range(1, n+1):
        print(f'{i}\t\t{"Fizz" * (i % 3==0)}{"Buzz" * (i % 5==0)}{"Baxx" * (i % 7==0)}')

fizzbuzz(1000)

File: RS/fizzbuzz.rs
fn fizz_buzz(n: u32, divisors: [(u32, &str); 3]) -> String {
    let mut base = String::from("");

    for i in 0..3 {
        if n % divisors[i].0 == 0 {
            base = base + divisors[i].1;
        }
    }
    return base;
}

fn main() {
    let d: [(u32, &str); 3] = [(3, "Fizz"), (5, "Buzz"), (7, "Baxx")];

    for i in 1..1001 {
        println!("{:04}   {}", i, fizz_buzz(i, d));
    }
}
